src/vulkan-renderer/application.cpp

```diff
@@ -575,6 +573,12 @@ void Application::run() {
         update_imgui_overlay();
         render_frame();
         process_mouse_input();
+        if (m_input_data->was_key_pressed_once(GLFW_KEY_N)) {
+            load_octree_geometry();
+            generate_octree_indices();
+            m_index_buffer->upload_data(m_octree_indices);
+            m_vertex_buffer->upload_data(m_octree_vertices);
+        }
```

Comment:
@aschaeffer once mentioned, that we should implement a layer between bindings and its calling commands. I cannot fully remember the conversation or the technical terms he used. Maybe @aschaeffer can further elaborate? I think we should implement that as soon as possible, but maybe not in this PR.

Reply:
Yes this goes into the direction of offering a good API for the renderer. That shouldn't be in this pull request, but there is already a ticket for that: https://github.com/inexorgame/vulkan-renderer/issues/297